tree/ntuple/v7/src/RNTuple.cxx

```diff
@@ -251,20 +238,20 @@ void ROOT::Experimental::RNTupleReader::Show(NTupleSize_t index, const ENTupleSh
    if (fModel)
       entry = fModel->GetDefaultEntry();
 
-   switch(format) {
+   switch (format) {
    case ENTupleShowFormat::kCompleteJSON:
       reader = GetDisplayReader();
       entry = reader->GetModel()->GetDefaultEntry();
       // Fall through
    case ENTupleShowFormat::kCurrentModelJSON:
       if (!entry) {
-         output << "{}" << std::endl;
-         break;
+         reader = GetDisplayReader();
+         entry = reader->GetModel()->GetDefaultEntry();
       }
```

Comment:
Alternatively, how about removing the `if` statement in lines 237-239 and directly initializing `entry` in line 236 as in
```c++
REntry *entry = GetModel()->GetDefaultEntry();
```

I think this is probably preferable, as `RNTupleReader::LoadEntry()` will initialize `fModel` from the ntuple descriptor anyways.
In the current state, a call to `Show()` followed by `LoadEntry()` triggers the generation of the model twice (once for `fDisplayReader.fModel`, and once for `fModel` itself).

Anything against proceeding this way?

Reply:
Ah, that makes sense. Then I suppose lines 247-250 can also be removed right?